### ISSUE TITLE
Instability from system restart

### ISSUE THREAD

**Issue by megatog615** [dekuNukem/USB4VC#13]
Sometimes when I reboot the machine(a socket 7 system), the USB4VC becomes either entirely unresponsive(no LED lighting from input) or seemingly responsive but the computer doesn't see any input.

I see there is a [technical bulletin](https://github.com/dekuNukem/USB4VC/blob/master/technical_bulletins.md) that is supposed to "improve stability" but, I don't want to go through with this modification without knowing what problem it specifically tackles(I'm not sure if it is relevant to this specific issue) and how it works.

**Comment by jamescadd** [comment]
Any update @dekuNukem ?  Would be nice to have an updated release with the last few commits to make sure it's not something that's already fixed.

**Comment by dekuNukem** [comment]
Hi there! can you let me know the model of the motherboard you're using? I personally haven't encountered an issue like this myself, would like to test it out.

**Comment by megatog615** [comment]
[BCM-SQ594](https://theretroweb.com/motherboards/s/bcm-sq594)

**Comment by megatog615** [comment]
I don't know why it took me so long but I just realized I could SSH into the thing and possibly monitor it on the command line. `dmesg` shows a bunch of `Undervoltage detected!` then `Voltage normalized` over and over again after using it for a while. My usb4vc is connected to a USB PD charger.

My Pi is a Model 3B Rev 1.2.

**Comment by megatog615** [comment]
I want to update this by mentioning that I've since found out that the Pi 3 had a history of pickiness with power supplies and the problem is most likely related to that. I'm not 100% sure if this should be closed because of that but maybe some mention of it in the readme?

**Comment by dekuNukem** [comment]
Yeah RPi in general are rather picky about the quality of USB power adapter.

I'll mention that in user guide, and I'll close this for now.
